demisto_sdk/commands/validate/validators/BC_validators/BC107_is_valid_toversion_on_modified.py

```diff
@@ -14,30 +16,111 @@
 
 class IsValidToversionOnModifiedValidator(BaseValidator[ContentTypes]):
     error_code = "BC107"
-    description = (
-        "Check that the toversion property was not changed on existing Content files."
-    )
+    description = "Check that the toversion property was not changed on existing Content files. unless there's a new file with the same id and continues from_version."
     rationale = "Changing the `toversion` field for a content item can break backward compatibility."
-    error_message = "Changing the maximal supported version field `toversion` is not allowed. Please undo, or request a force merge."
+    error_message = "Changing the maximal supported version field `toversion` is not allowed. unless you're adding new content_item with the same id {0} and their from/to version fulfills the following:\n{1}"
```

Comment:
remove {0}

Reply:
in this string it's replaced with the id, i removed the {0} we saw in the demo

Reply:
```suggestion
    error_message = "Setting a `toversion` field is not allowed, unless replaced by a new content item with the same ID {0}, and their fromversion/toversion fields fulfills the following:\n{1}"
```